services/venice-router/src/main/java/com/linkedin/venice/router/MetaDataHandler.java

```diff
@@ -175,6 +173,11 @@ public void channelRead0(ChannelHandlerContext ctx, HttpRequest req) throws IOEx
         // URI: /value_schema/{$storeName}/{$valueSchemaId} - Get single value schema
         handleValueSchemaLookup(ctx, helper);
         break;
+      case TYPE_LATEST_VALUE_SCHEMA:
```

Comment:
I am wondering if we can keep getLatestOrSupersetSchema as the API endpoint in the router side. This name would be more make sense to me. Same as the below logic method name.

I understand we don't want to change the API in SchemaFetcher interfaces to perform a smoother upgrade, but I think we can make it more clear on router side implementation. What do you think about this?
(but actually, if we want to bump venice version to achieve the new behavior, we can also change API though)

Reply:
The change I'm trying to make here is that "Latest schema" = superset schema if present or value schema with largest id

`/latest_or_superset_schema` doesn't look good IMO. I'd rather use `/<something>_schema`. Open to suggestions for what the `something` should be

Reply:
Hmm I don't have a very strong opinion towards the naming, but throughout the discussion with customers and other teammates I feel like "latest" alone might not be clear enough for them to illustrate the idea:
1. RC/WC enabled - superset schema / 2. WC disabled - the one with largest schema id.
customers tend to think getLatestSchema being the one with largest ID shown on Nuage... ( and some of them also think that schema must be the one registered by them because they don't know we also generate schema internally :) ) 

Anyway this is not a blocker, if we have enough doc about this I think it is fine for now.
(getSupersetOrLatestValueSchema() API is in SchemaRepo for internal use.)